_data/universities/utk.yml

```diff
@@ -0,0 +1,3 @@
+name: University of Tennessee, Knoxville
+personnel:
+  - j-s-ashley
```

Comment:
The `_data/` directory is for meant for institute researchers/staff, so this isn't needed for Fellows (and I think the CI is throwing an error as it is expecting additional files to exist given this university file) so this can get deleted.

```suggestion
```